src/services/itemMembership/plugins/MembershipRequest/repository.ts

```diff
@@ -7,4 +7,30 @@ export class MembershipRequestRepository extends AbstractRepository<MembershipRe
   constructor(manager?: EntityManager) {
     super(MembershipRequest, manager);
   }
+
+  async get(memberId: string, itemId: string) {
+    return await this.repository.findOne({
```

Comment:
I think you have to check that memberId and itemId are defined to avoid TypeORM to returns all the entries.